docs/developer/reference/services/buildbot.rst

```diff
@@ -147,4 +147,7 @@ Monitoring
 
 Buildbot runs are logged at `Launchpad
 Buildbot <https://chat.canonical.com/canonical/channels/launchpad-buildbot>`__
-channel on Mattermost. 
\ No newline at end of file
+channel on Mattermost. If a build fails, you can navigate to the build log
+of your build and review the summary to identify the cause of the failure.
+If the failure is not related to your changes, you can `restart the 
```

Comment:
This is hard to know, I would just say "if you suspect the build is failing due to flakiness of the test infrastructure and not your changes"

Reply:
+1 What Yuliy says